pkg/config/reader.go

```diff
@@ -72,13 +72,20 @@ func (r *FileReader) parseConfig() error {
 		return nil
 	}
 
-	usedConfigFile, err := fsutils.ShortestRelPath(usedConfigFile, "")
-	if err != nil {
-		r.log.Warnf("Can't pretty print config file path: %s", err)
+	if usedConfigFile == os.Stdin.Name() {
+		usedConfigFile = ""
```

Comment:
Since this will be used in the log `Used config file`, would it make sense to either do conditional logging (not a file if stdin) or just set this to `stdin` so the log says `Reading config file stdin`?

Reply:
I added a log.